test/lib/utils/addresses.ts

```diff
@@ -25,6 +25,27 @@ export const TOKENS: Record<number, Record<string, TestToken>> = {
             decimals: 6,
             slot: 2,
         },
+        swETH_bb_a_WETH_BPT: {
+            address: '0x156c02f3f7fef64a3a9d80ccf7085f23cce91d76',
+            decimals: 18,
+            slot: 0,
+        },
```

Comment:
Given this has the same content as the pool, we could avoid duplicates by using the pool only - what do you think?